browser/src/index.tsx

```diff
@@ -256,7 +245,28 @@ const start = (args: string[]) => {
         pendingTimeout = null
     }
 
-    instance.setFont(Config.getValue<string>("editor.fontFamily"), Config.getValue<string>("editor.fontSize"))
+    const configChange = () => {
+        cursorLine = Config.getValue<boolean>("editor.cursorLine")
+        cursorColumn = Config.getValue<boolean>("editor.cursorColumn")
+        UI.setCursorLineOpacity(Config.getValue<number>("editor.cursorLineOpacity"))
+        UI.setCursorColumnOpacity(Config.getValue<number>("editor.cursorColumnOpacity"))
+
+        if (cursorLine) {
+            UI.showCursorLine()
+        }
+
+        if (cursorColumn) {
+            UI.showCursorColumn()
+        }
+
+        remote.getCurrentWindow().setFullScreen(Config.getValue<boolean>("editor.fullScreenOnStart"))
+        instance.setFont(Config.getValue<string>("editor.fontFamily"), Config.getValue<string>("editor.fontSize"))
+        renderFunction()
```

Comment:
I don't think we need to call renderFunction explicitly (as it is already called by `requestAnimationFrame`, but I don't think this will hurt either).